pkg/gadgettracermanager/containers-map/tracer.go

```diff
@@ -75,6 +77,7 @@ func NewContainersMap(pinPath string) (*ContainersMap, error) {
 		return nil, fmt.Errorf("failed to load asset: %w", err)
 	}
 
+	// TODO: can we just defer coll.Close()?
```

Comment:
We can't: it would close all the programs and maps.

Reply:
I think the map would still exist on the bpffs but the fd would be closed in the process. And with #654 we will need to keep it open to pass it around to other gadgets.

Reply:
You're right. I'll remove this comment then.